common/culture/traditions/00_regional_traditions.txt

```diff
@@ -476,26 +502,39 @@ tradition_horse_lords = {
 
 	is_shown = {
 		OR = {
-			like_mongolic_group_trigger = yes
-			like_turkic_group_trigger = yes
+			# Warcraft
+			has_cultural_pillar = heritage_yaungolic
+			has_cultural_pillar = heritage_zaetaric
+			has_cultural_pillar = heritage_tharian
+			has_cultural_pillar = heritage_cenaric
+			has_cultural_pillar = heritage_orcish
+			has_cultural_pillar = heritage_agamagganic
 		}
 	}
 	
 	can_pick = {
-		custom_description = {
-			text = culture_in_steppe_desc
-			always = no
+		# Warcraft
+		custom_tooltip = {
+			text = culture_in_plains_desc
 			any_culture_county = {
 				percent >= 0.4
-				any_county_province = {
-					steppe_region_trigger = yes
+				OR = {
+					any_county_province = { steppe_region_trigger = yes	}
+					any_county_province = {
+						OR = {
+							geographical_region = world_innovation_camels
+							geographical_region = custom_oasis
+							geographical_region = custom_drylands
+						}	
+					}
 				}
 			}
 		}
 	}
 	
 	parameters = {
-		unlock_maa_horse_archers = yes
+		# Warcraft
+		# unlock_maa_horse_archers = yes
```

Comment:
Hmm. Centaurs may need it badly. I would do something about it.

Reply:
`unlock_maa_horse_archers ` points to `horse_archers` in vanilla file `00_cultural_maa_types.txt` but `horse_archers` is missing in the GoA version, therefore I removed the trigger to prevent an error. The spreadsheet lists Marauders as the only Centaur specific MaA implemented right now and they are classed as Light Cavalry. Any insight @Gaben1025 ?

Reply:
Fair for now. Maybe need to redesign horse archers later.
Resolve conversation as you read.